sites/fast-website/src/app/index.ts

```diff
@@ -1,4 +1,13 @@
+import { DesignSystem } from "@microsoft/fast-foundation";
+import * as components from "@microsoft/fast-components/dist/esm/custom-elements";
+import { siteColorSwatch } from "./components";
 import "./css/style.css";
+DesignSystem.getOrCreate().register(
+    ...Object.values(components).map(x => (x as () => void)()),
+    siteColorSwatch({ prefix: "site" })
```

Comment:
@EisenbergEffect I found a bit of an odd use-case we might want to address. The site's color swatch extends FAST's Radio component, which is a Foundation component. Migrating that straight over to vNext infra causes the component to blow up because foundation is looking for a presentation in the Design System, even though the component is actually being defined via `@customElement`. I'm wondering if there is a way to make this case easier, where an author wants to *extend* a component with a `FoundationElement` base-class and define it using `@customElement`

Reply:
I think we could fix this pretty easily. In FoundationElement, if there is no. presentation object present, just do nothing. Then the base connect implementation should kick in and look up the template and the styles with the definition object that comes from the decorator. Does that sound right?

Reply:
Actually, it's more tricky than that. We need to change the way the presentation object is looked up internally, but we could probably make that work. How fast do we need this do you think? I might be able to do that on Friday.

Reply:
Not fast, just a case I encountered here and I'm sure it will happen again. I'll write up an issue to track.

Reply:
I had another look at the code and I think it's just a couple of small changes. I'll see if I can get my PR-for-the-week in tomorrow to enable this 🤣

Reply:
nice, if not I can look into it next week. https://github.com/microsoft/fast/issues/4794 is tracking